### PR TITLE
NEW Adding a loading indicator

### LINKED ISSUE #201: Registries cannot be previewed before they are published
Currently when you attempt to preview a registry before it's published there is a 404 on the AJAX request to get the schema to initialise the React app. This just needs the `stage=Stage` param on it.

### LINKED ISSUE #202: Replace "Loading..." with a spinner
Currently while loading a CKAN page we have some text ("Loading...") come up while it's loading. This is pretty bad UX and can be improved by adding a loading indicator.

### PR DESCRIPTION
**Pull request by ScopeyNZ**
Fixes #202

This PR also includes a quick fix for #201

Fixes #201

### CODE REVIEW

File: client/src/components/CKANExampleApp.js [hunk 1]
```diff
@@ -29,7 +30,9 @@ class CKANExampleApp extends Component {
    * When the component has mounted, fetch the client schema configuration
    */
   componentDidMount() {
-    const schemaPath = `${window.location.pathname}/schema`;
+    const params = new URLSearchParams(window.location.search || '');
+    const stage = params.has('stage') ? `?stage=${params.get('stage')}` : '';
+    const schemaPath = `${window.location.pathname}/schema${stage}`;
```
Comment (ScopeyNZ) [inline client/src/components/CKANExampleApp.js]: Fix for #201 . Everything else is cleaning up the loader

Comment (robbieaverill) [reply client/src/components/CKANExampleApp.js]: Would this also need to include things like the Fluent locale or Subsite ID, which may be in the URL params when previewing in the CMS?

Comment (ScopeyNZ) [reply client/src/components/CKANExampleApp.js]: Possibly? I'm not sure how it's handled when those are omitted? Any advice here?

Comment (robbieaverill) [reply client/src/components/CKANExampleApp.js]: You could pass through all URL params maybe. Otherwise I dunno. This is good for now

**Pull request merged**